src/appsCreate.ts

```diff
@@ -0,0 +1,145 @@
+import * as api from "./api";
+import * as FirebaseError from "./error";
+import * as logger from "./logger";
+import { pollOperation } from "./operation-poller";
+
+const ONE_SECOND_MILLIS = 1000;
+
+export enum AppPlatform {
+  IOS = "IOS",
+  ANDROID = "ANDROID",
+  WEB = "WEB",
+}
+
+export interface ShaCertificate {
+  name?: string;
+  certType: "SHA_1" | "SHA_256";
+  shaHash: string;
+}
+
+/**
+ * Send an API request to create a new Firebase iOS app and poll the LRO to get the new app
+ * information.
+ * @return a promise that resolves to the new iOS app information
+ */
+export async function createIosApp(
+  projectId: string,
+  options: { displayName?: string; bundleId: string }
+): Promise<any> {
+  try {
+    const response = await api.request("POST", `/v1beta1/projects/${projectId}/iosApps`, {
+      auth: true,
+      origin: api.firebaseApiOrigin,
+      timeout: 15 * ONE_SECOND_MILLIS,
```

Comment:
Is there a standard library you could use to convert seconds to milliseconds?

Reply:
I don't think we have such a method. I think I will just use `...TIMEOUT_MILLIS` here

Reply:
Done